wiki/Tournaments/THOM_CUP/1/en.md

```diff
@@ -0,0 +1,190 @@
+---
+tags:
+- THOM
+- THOM CUP
+---
+THOM CUP \#1
+======================
+
+![THOM CUP Logo](logo.png)
+
+**THOM CUP \#1** (short for **"The Heptakey osu!mania Cup"**) is a one-on-one euro-american osu!mania tournament. It is the 1st installment of the THOM CUP. More specifically, this tournament is an osu!mania 7-key tournament.
```

Comment:
`**THOM CUP \#1** (short for **"The Heptakey osu!mania Cup"**) ->  **The Heptakey osu!mania Cup \#1** (***THOM CUP \#1***)`